### DISCUSSION TITLE
Can i run multi instances on binance exchange (or with other bots not nautilus related)?

### DISCUSSION THREAD

**Discussion by gaugau3000** [nautechsystems/nautilus_trader#1119] [Q&A] [answered]
I,

From far I remember (a few monthes ago) when I received orders from binance order web sockets not related to the current instance, nautilus was not able to manage it filter correctly. Can you tell me more about it  : does a UUID is attached to order like clientOrderId in binance to ensure only instance coming from the instance this take into account (or other mechanism)?

Many tks.

**Comment by cjdsellers** [comment, marked as answer]
Hi @gaugau3000 

Yes, there have been some improvements. This should be managed using a combination of the `external_order_claims` and `filter_unclaimed_external_orders` config options. It would be best to search for these in the documentation.

**Reply by gaugau3000** [reply]
Many tks I will take a look !
